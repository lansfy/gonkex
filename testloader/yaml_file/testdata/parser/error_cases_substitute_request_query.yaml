- name: loader MUST fail if cannot substitute requestArgs to request query
  method: POST
  path: /some/path
  query: "?name={{ .foo }"
  request: "requestbody"
  response:
    200: "somebody"
  cases:
    - requestArgs:
        foo: 'value 1'
        bar: other1
