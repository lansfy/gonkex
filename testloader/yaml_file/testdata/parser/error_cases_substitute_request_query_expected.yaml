- Error: 'preprocess file testdata/parser/error_cases_substitute_request_query.yaml: template: :1: unexpected "}" in operand'
